virttest/virsh.py

```diff
@@ -4857,3 +4857,37 @@ def guestinfo(name, options="", **dargs):
     :return: CmdResult object.
     """
     return command("guestinfo %s %s" % (name, options), **dargs)
+
+
+def get_user_sshkeys(name, user, **dargs):
+    """
+    list authorized SSH keys for given user (via agent)
+
+    :param name: VM name
+    :param user: user to list authorized keys for
+    :param dargs: standardized virsh function API keywords
+    :return: CmdResult object.
+    """
+    return command("get-user-sshkeys %s %s" % (name, user), **dargs)
+
+
+def set_user_sshkeys(name, user, options="", key_file=None, **dargs):
+    """
+    manipulate authorized SSH keys file for given user (via agent)
+
+    :param name: VM name
+    :param user: user to set authorized keys for
+    :param options: options of get-user-sshkeys command
+    :param key_file: optional file to read keys from
+    :param dargs: standardized virsh function API keywords
+    :return: CmdResult object.
+
+    """
+    cmd = "set-user-sshkeys %s %s " % (name, user)
+    if options == "--reset":
+        cmd += "--reset "
+    if options == "--remove":
```

Comment:
If options is --remove, is key_file necessary to be provided
If this is case, better change to cmd += "--remove  --file %s" % key_file

Reply:
--file can be omitted